src/keyutil/keyutil.c

```diff
@@ -39,6 +42,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "\n%3$sCommands:%4$s\n"
                "  validate               Load and validate the given certificate and private key\n"
                "  public                 Extract a public key\n"
+               "  pkcs1-to-pkcs7 [INPUT] Convert a PKCS1 signature to a PKCS7 signature\n"
```

Comment:
I think it's a bit more future proof to name the verb `pkcs7` and add an option `--signature` that takes the signature to attach to the PKCS#7 signature. Specifically, I'm thinking at some point we'll want to extend this to create a non-detached PKCS#7 signature, which we'd probably do by allowing to pass a payload with `--payload` or so.